xml2rfc/writers/doc.py

```diff
@@ -149,7 +149,8 @@ def process(self):
             # This depend on at least one generic option with title being listed before '--config-file'
             if a.container.title == 'Generic Options with Values':
                 genargs_group = a.container
-            if a.container.title == 'optional arguments':
+            if a.container.title == 'optional arguments' or a.container.title == 'options':
+                # Python 3.10 calls this options while Python 3.6 to Python 3.8 calls this optional arguments
```

Comment:
what does 3.9 do?

Reply:
This was changed in Python 3.10: https://docs.python.org/3.10/whatsnew/3.10.html#argparse
Sorry, I missed this before the merge.

Reply:
I have updated the comment.